pa/biGraph.ml

```diff
@@ -0,0 +1,176 @@
+open Pervasives
+open BatPervasives
+open Apak
+
+include Log.Make(struct let name = "BiGraph" end)
+
+module type Vertex = sig
+  type t
+  val hash : t -> int
+  val equal : t -> t -> bool
+  val compare : t -> t -> int
+  val pp : Format.formatter -> t -> unit
+end
+
+module type S = sig
```

Comment:
This signature is unused.  If you want Make to be associated with this signature, the best way to do it is to create a separate .mli file (and add comments for top-level functions to this file).